src/phases/query/CollScanComplexPredicate.yml

```diff
@@ -20,6 +20,19 @@ GlobalDefaults:
     length: {^RandomInt: {min: 50, max: 250}}, alphabet: "0123456789"
   }}
 
+  LargeIn: &LargeIn {$in: {^Array: {
+    of: &IntOrString {^Choose: {from: [
+        {^RandomInt: {min: 0, max: &Billion 1_000_000_000}},
+        *RandomLongStringValue
+    ]}},
+    number: 1000,
+  }}}
+
+  VeryLargeIn: &VeryLargeIn {$in: {^Array: {
+    of: *IntOrString,
+    number: 10000,
```

Comment:
I'd be surprised if this successfully executes on the large version of this workload, given that 100k docs was the limit for the microbenchmark workloads. Same concern for the other workload as well.

Reply:
I was expecting it to fail at least on some variant as well but it didn't ([patch](https://spruce.mongodb.com/version/65a81f7c3627e0d7d5f5d4b4/tasks?page=0&sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC&taskName=coll_scan)). I'll run a couple more patches tomorrow to confirm it doesn't randomly fail.